src/components/dialog/Dialog.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import {ArgsTable, Meta, Story, Canvas} from '@storybook/addon-docs';
+
+import DialogA11y from './stories/Dialog.a11y.mdx';
+import Dialog from './Dialog';
+import {Default} from './stories/stories';
```

Comment:
Could you move it also to one mdx?

Reply:
let's decide today or on friday, considering our days off in this week